janus/lib/server.rb

```diff
@@ -42,6 +40,8 @@ class Server < Etna::Server
 
     post '/update_key', action: 'user#update_key'
 
+    get '/:project_name', action: 'admin#project', auth: { user: { can_edit?: :project_name } }
+
```

Comment:
This (et al) provides consistent routing across apps for getting to a project's main page.